### PR TITLE
fix: validator table ui issues

### PR DESCRIPTION
**Pull request by nimit9**
Closes: #6995 

Fixed validator table UI issues

@IrakliJani Can you please explain the 3rd pointer, how is the content not proportional, and what is the desired ui? I'll make the changes to fix that issue and push it
Fix Validaor Table UI issues

### CODE REVIEW

**Comment by IrakliJani** [comment]
hi @nimit9 sorry for getting back so late on this. the team was on the offsite previous week


CleanShot 2023-09-18 at 13 35 59@2x

adding a screenshot to highlight what I meant in point 3 (see arrows)
hope this is clearer now

**Comment by nimit9** [comment]
Hey @IrakliJani, does this look fine now?

Screenshot from 2023-09-19 00-05-47

**Comment by IrakliJani** [comment]
@nimit9 yup, way better. is the PR ready to be reviewed?

**Comment by nimit9** [comment]
> @nimit9 yup, way better. is the PR ready to be reviewed?

Yes @IrakliJani , it's ready to be reviewed.

**Comment by IrakliJani** [comment]
thanks @nimit9 I'll have a look now

File: packages/frontend/src/components/SettingsValidator/index.tsx [hunk 1]
```diff
@@ -47,7 +48,9 @@ export const SettingsValidator: FC<{ projectUuid: string }> = ({
                 >
                     <Text fw={500} fz="xs" c="gray.6">
                         {!!data?.length
-                            ? `Last validated at: ${data[0].createdAt}`
+                            ? `Last validated at: ${moment(
+                                  data[0].createdAt,
+                              ).format('MMMM Do YYYY, h:mm:ss a')}`
```
Comment (IrakliJani) [inline packages/frontend/src/components/SettingsValidator/index.tsx:53]: there is a formatTime function in schedulers view which you can either duplicate or reuse for consistency
```
export const formatTime = (date: Date) =>
    dayjs(date).format('YYYY/MM/DD hh:mm A');
```

**Review by IrakliJani — COMMENTED** [review]
code looks good, there's one comment that I left about date formatting. im testing locally now

File: packages/frontend/src/components/SettingsValidator/index.tsx (hunk 1 above)
Comment (nimit9) [reply packages/frontend/src/components/SettingsValidator/index.tsx:53]: Sure, will make this change

**Comment by IrakliJani** [comment]
tested locally, looks great @nimit9, waiting for e2e to pass before merging

**Comment by IrakliJani** [comment]
ignoring date tests, they are timeouts.

well done @nimit9 thanks for contributing! 🙌

**Pull request merged**